cosmwasm/polytone/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[workspace]
+members = ["packages/*", "contracts/accessories/*", "contracts/main/*"]
+
+[workspace.package]
+edition = "2021"
+license = "BSD-3-Clause"
+repository = "https://github.com/cosmos/ibc-apps/cosmwasm/polytone"
+rust-version = "1.67"
+version = "1.0.2"
+
+[profile.release]
+codegen-units = 1
+debug = false
+debug-assertions = false
+incremental = false
+lto = true
+opt-level = 3
+overflow-checks = true   # very important, do not turn these off.
+panic = 'abort'
+rpath = false
+
+[workspace.dependencies]
+polytone = { path = "packages/polytone", version = "1.0.0" }
+polytone-proxy = { path = "contracts/main/proxy", version = "1.0.0" }
+polytone-note = { path = "contracts/main/note", version = "1.0.0" }
+polytone-voice = { path = "contracts/main/voice", version = "1.0.0" }
+
+cosmwasm-schema = "1.2.1"
+cosmwasm-std = { version = "1.5.0", features = ["ibc3"] }
```

Comment:
Make sure you have the latest upstream, we bumped some dependencies: https://github.com/DA0-DA0/polytone/pull/63#pullrequestreview-2077612554

Reply:
also ref: https://github.com/DA0-DA0/polytone/pull/65

Reply:
Pulled in all at b459f279c07b48316a75b2b5d6bbdb7030b3a6d0. Excludes the releases & audit dep workflows atm. Will test those outside this repo to ensure only tags for polytone get released for.